frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -25,6 +25,10 @@ export const getDesiredClusterVersion = (cv: ClusterVersionKind): string => {
   return _.get(cv, 'status.desired.version');
 };
 
+export const getLastCompletedClusterVersion = (cv: ClusterVersionKind): string => {
+  return _.get(cv, 'status.history[0].version');
```

Comment:
This logic is wrong. You have to find the first item in the history that is completed.

```suggestion
  const history: UpdateHistory[] = _.get(cv, 'status.history', []);
  const lastCompleted: UpdateHistory = history.find(update => update.state === 'Completed');
  return lastCompleted && lastCompleted.version;
```